server/routes/referrals/riskInformationView.ts

```diff
@@ -23,13 +24,43 @@ export default class RiskInformationView {
     }
   }
 
+  roshAnalysisTableArgs(tagMacro: (args: TagArgs) => string): TableArgs {
+    return {
+      head: this.presenter.roshAnalysisHeaders.map((header: string) => {
+        return { text: header }
+      }),
+      rows: this.presenter.roshAnalysisRows.map(row => {
+        return [
+          { text: utils.convertToProperCase(row.riskTo) },
+          { text: tagMacro({ text: row.riskScore, classes: this.tagClassForRiskScore(row.riskScore) }) },
```

Comment:
(minor suggestion): Perhaps can reuse PresenterUtils.complexityLevelTagArgs?
Seems that this can be considered a UI component independent of underlying field use (risk vs complexity)?